kubernetes/ingress.go

```diff
@@ -59,6 +59,7 @@ type ingressLoadBalancer struct {
 }
 
 const (
+	ingressALBIPAdressType          = "alb.ingress.kubernetes.io/ip-address-type"
```

Comment:
Please change to `zalando.org/ip-address-type`, because annotations are specific to the tool.